src/mac/region/RegionCommon.c

```diff
@@ -202,7 +202,8 @@ TimerTime_t RegionCommonUpdateBandTimeOff( bool joined, bool dutyCycle, Band_t*
             }
         }
     }
-    return nextTxDelay;
+
+    return nextTxDelay==TIMERTIME_T_MAX?0:nextTxDelay;

```

Comment:
Could you please update your pull request to be compatible to the code conventions.

Reply:
We would expect:

`return ( nextTxDelay == TIMERTIME_T_MAX ) ? 0 : nextTxDelay; `

Reply:
Done!